README.md

```diff
@@ -25,6 +25,10 @@ _Windows:_ `iwr -useb https://get.allero.io/windows_install.ps1 | iex`
 
 
 ### 2. Validate your pipelines
+Local directory - a direct validate:
+* Validate your pipelines located under a path: `allero validate {path}`
+
+Remote directory - fetch your data and then validate:
```

Comment:
It's not a remote directory. It's remote source control.
Change it to "#### Validate pipelines from a remote source control"